bugsnag-plugin-android-anr/src/main/jni/bugsnag_anr.c

```diff
@@ -8,19 +8,15 @@ extern "C" {
 #endif
 
 JNIEXPORT void JNICALL Java_com_bugsnag_android_AnrPlugin_enableAnrReporting(
-        JNIEnv *env, jobject _this, jobject anr_buffer) {
-    // only install if provided a buffer
-    if (!(*env)->IsSameObject(env, anr_buffer, NULL)) {
-        bsg_handler_install_anr((*env)->GetDirectBufferAddress(env, anr_buffer));
-    }
+        JNIEnv *env, jobject _this, jobject plugin) {
+    bsg_handler_install_anr(env, plugin);
```

Comment:
I suspect `_this` and `plugin` are the same object. Passing `this` manually should not be necessary.

Reply:
This appears to be the case, I've amended the PR.